experiment/build/docker_images.py

```diff
@@ -50,6 +55,11 @@ def get_images_to_build(fuzzers, benchmarks):
     for fuzzer in fuzzers:
         for benchmark in benchmarks:
             for name_templ, obj_templ in templates.items():
+                if 'coverage' in fuzzer and 'runner' in name_templ:
+                    continue
```

Comment:
consider using `else` instead of continue

Reply:
Do you mean `if <condition>: pass; else: <action>;` ?

Reply:
ah i realized we aren't doing anything in this branch -- sorry, never mind